pkgs/development/tools/gomod2nix/builder/default.nix

```diff
@@ -0,0 +1,387 @@
+{ stdenv
```

Comment:
Once this PR is merged this file will need to kept in sync with the other `go` builders`. i.e changes will be applied to all three at once in this repo. Are you happy coping changes from here to the upstream repo or would it be easier to just remove it from the upstream repo so they don't drift apart?

Reply:
I want to give it some time, but the intention is to drop the builder code from the upstream repo eventually.

Reply:
For now I'm happy copying any changes.